### PR TITLE
fix(learn) formatting issue with else-if statements tests

### LINKED ISSUE #39538: [BUG] - Formatting required in `Introducing Else If Statements`
**Describe your problem and how to reproduce it:**

The tests appear to require specific (albeit conventionally correct) formatting in order for the code to pass. This code block, provided by a user, did not pass:
```js
function testElseIf(val) {
if (val > 10) {
  return "Greater than 10";
} else  if (val < 5) {
  return "Smaller than 5";
} else {
  return "Between 5 and 10";
}
}

testElseIf(7);
```

However, if I ran the formatter with `Ctrl`+`Alt`+`F`, the resulting formatted code *did* pass:
```js
function testElseIf(val) {
  if (val > 10) {
    return "Greater than 10";
  } else if (val < 5) {
    return "Smaller than 5";
  } else {
    return "Between 5 and 10";
  }
}

testElseIf(7);
```

**Add a Link to the page with the problem:**

[Basic JavaScript - Introducing Else If Statements](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/introducing-else-if-statements)

**Tell us about your browser and operating system:**
* Browser Name: Chrome
* Browser Version: 85.0.4183.83
* Operating System: Windows 10 v1909

### PR DESCRIPTION
**Pull request by siddharthpk**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39538

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM! 🥳 Congratulations on opening your first PR with us! As a quick reminder, our workflow requires at least one additional approving review from the team before this gets merged. 

@Sky020 - I believe this was a fix you proposed in the linked issue. Would you like to take a look? I tested it locally with the unformatted code and it did not erroneously fail the new test.

**Review by ShaunSHamilton — APPROVED** [review]
Hi @siddharthpk . Congrats, on your first pull request (PR)! 🎉

Thank you, for your contribution! 👍
We are happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**
